catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/DeleteOperations.java

```diff
@@ -466,4 +544,26 @@ private boolean isStorageAvailable(StorageProvider storageProvider) {
         }
         return true;
     }
+
+    private void logFailedQueryInfo(DeleteRequest deleteRequest, String attributeName,
+            QueryResponse query) {
+        if (LOGGER.isDebugEnabled()) {
+            LOGGER.debug(
+                    "While rewriting the query, did not get a metacardId corresponding to every attribute."
+                            + "Original Delete By attribute was: " + attributeName);
+            LOGGER.debug("Metacards unable to get Metacard ID from are: "
+                    + deleteRequest.getAttributeValues()
```

Comment:
Pull this out into a string and use param interpolation. The concatenation, especially with the inline stream processing, just looks terrible.